Kernel/Time/HardwareTimer.h

```diff
@@ -14,18 +14,13 @@
 namespace Kernel {
 
 enum class HardwareTimerType {
-#if ARCH(X86_64)
-    i8253 = 0x1,                   /* PIT */
-    RTC = 0x2,                     /* Real Time Clock */
-    HighPrecisionEventTimer = 0x3, /* also known as IA-PC HPET */
-    LocalAPICTimer = 0x4,          /* Local APIC */
-#elif ARCH(AARCH64)
-    RPiTimer = 0x5,
-#elif ARCH(RISCV64)
-    RISCVTimer = 0x6,
-#else
-#    error Unknown architecture
-#endif
+    i8253,                   // PIT
```

Comment:
Should we at least have comments with sections like

```
  // X86
  i8253,
  RTC, ...
  // AArch64
  ...
```

Why do the values have to be unique across ISAs btw? Could each ISA just start at 1 and increase? That'd also fix the relabeling problem, right?

(Either's fine.)

Reply:
I prefer having less `#if ARCH`s if possible, so I went with the comments.